boards/kernel_layout.ld

```diff
@@ -155,10 +160,9 @@ SECTIONS
         /* Optional .app sections a convenience mechanism to bundle tock
            kernel and apps into a single image */
         KEEP (*(.app.*))
+        _eapps = PROG_ORIGIN + PROG_LENGTH - 1;
```

Comment:
Is this symbol used for something (eventual page-locking of flash?)

Currently it's set to the maximum amount of flash the linker script will allow for apps, but that's a bit misleading as it's not actually where loaded app memory ends, Tock can only establish that dynamically (or if tockloader wrote a variable somewhere when it loaded apps, but that feels too black magic-y). Given that, is this symbol really useful?